.hgsigs

```diff
@@ -3,3 +3,8 @@ be98370f35044c24e60f75102ff26091d4614864 0 iQEcBAABAgAGBQJWyH6YAAoJEKMv031G/7TTm
 038bde8cbc27552c1293aa7020e6eb39236f2209 0 iQEcBAABAgAGBQJWyKd4AAoJEKMv031G/7TT24EH/1D3Pc7e0py3W7nkvu569xaqQfZje62ipMahZPi4JPSBuMWZpQnaoT1e9NASsiWgJQtsPj9/heNn05sT7Pwxcg0G1VeilZqpjKmvgVGt3txeus+IJVQ0X4cpe8FY04+VrQ1SkCyncZbeQTMU7b5504ic8hr4F3F7HN6IW7Kms+2hEgnEOkrJJkjPO8TJZ1OFgowSQSKA8QbJKI7Fp+dNyMpj42TM+0zcGk4e23yKYsQix085YM6tNgLA+yHg6nhBg5niW7Anzr6DMjyYRDrSiveZMPkC/0194wiDz3UWfW9qsoroJUFIpAmcwwpPWakZ3WPZ08RrJLo+YIEU6NbYLAg=
 cd12c2e21dad7034b6775e3ed3d2d8be1ab46a73 0 iQEcBAABAgAGBQJW4eAhAAoJEKMv031G/7TTW0cH/jqSHwWFsjnynliXCnXY5W+HBUYQ2+KIJN7swAgjS4Wjn5OBrbKxg6TNtlCiCsMAIGvgXFvD+G8R6wtVziL8is6cyzK++tLdFUUOlfQJuvxAhCMYJlf+qKwuhoXH0M/m9Ff0+AFgaBdPLSDLWxIHLM+nWdKFMBlbpx+Pbt9JLz2CNWJax4+ULsXqFQUd++rcL3WqRWHROFiP6X9XqrpNwET6bUHyvg5F9CMO+5JxKuWnIYN1/zJ5j3rZi4vgjNPUgkgWWbHHtuUzHFP2IJfj+jI4OcRZoLv/i4Mjk1xJAWB61P0AWfhkgGK9XpTrZP0yqURVVmDnyMvtB8OTpRiiXnc=
 e3c31f3768f3a5a5cc0f88201d3b93d4fa3d9245 0 iQEcBAABAgAGBQJXMv+PAAoJEKMv031G/7TTji4H/3UNRufP7wvmmeUBFB59s+atvG6TSeH3twUe9+x2+jpFBXt8E96fDWYD1kcuOf2rSYQ0JeN+munOjjhQdldOBUIt8C5ww+Y1HYq53QaA9QiUF7IAMj1tRYQ+rtF1/5RfGCMHGWIeCTIx/ckHnnvnLLeaCVPIvQzoN1ATGmOe/1Ojz10Aze+2Wk6evAYJv4WqUWCai/zugIZIskfRNFrVKXOYNF2kTPogOFIPO0Q7K0anknl1HszeW+U/vJXRoWXVzgRMexg/G+i8MRfsySL265FWD/J1u3DtC3rTw/khDJBImdVMR2vnQu/W4p6MlUxSYKUAgQsLizaASJ8KyVVPhcA=
+76c0e45f2beed9a7cfd4a04aa271554a10a7b8df 0 iQEcBAABAgAGBQJX0WZ6AAoJEKMv031G/7TTT/QIAL8bbkMB4cenAJMfX0r0Q+OinkfTu4TYjz3HvM91aookeF2bDHF0vwYnQvqH1IxL3mK4fMcks9RTcGTl2CLZix6iSMDWeYLMWOK1qHFncvP/Ff9rDZZvOxwqABRGPO2bv/6mIG0Agr6aIxt3iwetWq0v++mwvsBQ7DTO9oOXbKhVfYx+/KYdWekcSeBIrxI6znXoR3IOsYH/7rKKJK1Pf/v7QBUD9nrDt3hbUhvWpdZElJaLr6/dECZ2Afaaii5BfgNB1kmc0nQkR2AkvKIVMnt7dXQkKomxkCepFfTT+s6IyAkx4x8h87OV/Y0HIZB3KudzSOcWj6V5H+OJXvvVChM=
```

Comment:
because the git repo is the secondary repository while the primary one lives on Freenet.